packages/dashboard/examples/ed-visits-county-file.json

```diff
@@ -1,712 +1,618 @@
-                    "filtered-text1701791173259",
-                    "filtered-text1701791674571",
-                    "filtered-text1701791683990",
-                    "filtered-text1701888972205",
-                    "filtered-text1701888980655"
-                ],
-                "values": [
-                    "Alabama",
-                    "Alaska",
-                    "Arizona",
-                    "Arkansas",
-                    "California",
-                    "Colorado",
-                    "Connecticut",
-                    "Delaware",
-                    "District of Columbia",
-                    "Florida",
-                    "Georgia",
-                    "Hawaii",
-                    "Idaho",
-                    "Illinois",
-                    "Indiana",
-                    "Iowa",
-                    "Kansas",
-                    "Kentucky",
-                    "Louisiana",
-                    "Maine",
-                    "Maryland",
-                    "Massachusetts",
-                    "Michigan",
-                    "Minnesota",
-                    "Mississippi",
-                    "Montana",
-                    "Nebraska",
-                    "Nevada",
-                    "New Jersey",
-                    "New Mexico",
-                    "New York",
-                    "North Carolina",
-                    "North Dakota",
-                    "Ohio",
-                    "Oklahoma",
-                    "Oregon",
-                    "Pennsylvania",
-                    "Rhode Island",
-                    "South Carolina",
-                    "Tennessee",
-                    "Texas",
-                    "United States",
-                    "Utah",
-                    "Vermont",
-                    "Virginia",
-                    "Washington",
-                    "West Virginia",
-                    "Wisconsin",
-                    "Wyoming"
-                ],
-                "active": "United States",
-                "tier": 1
-            }
-        ]
-    },
-    "rows": [
-        [
-            {
-                "width": 12,
-                "widget": "filter-dropdowns1701792379133"
-            },
-            {
-                "equalHeight": false
-            },
-            {},
-            {}
-        ],
-        [
-            {
-                "width": 6,
-                "widget": "filtered-text1701888972205"
-            },
-            {
-                "equalHeight": false,
-                "width": 6,
-                "widget": "filtered-text1701888980655"
-            },
-            {
-                "width": null
-            },
-            {
-                "width": null
-            }
-        ],
-        [
-            {
-                "width": 12,
-                "widget": "chart1701791122773"
-            },
-            {
-                "equalHeight": false
-            },
-            {},
-            {}
+  "dashboard": {
```

Comment:
@mpallansch do you have some eslint config that's different from the code base? It reformatted this after I saved.

Reply:
Not that I'm aware of, I don't think I've had this issue before